domains/POAS/events/attorney-started/examples/example.json

```diff
@@ -0,0 +1,4 @@
+{
+  "uid": "M-1234-5678-9012",
+  "actorUID": "urn:opg:poas:makeregister:users:740e5834-3a29-46b4-9a6f-16142fde533a"
```

Comment:
Some considerations for the future:

Would it be better to include a list of actorUIDs rather than firing multiple of the same event per attorney?

Do we always request signatures from all attorneys? If we are, do we even need a list of the actorUids? 🤔

Reply:
Good questions - like we discussed, lets leave it as an event per actor for now as it does give us some flexibility with replaying events after a failure but we may decide the likelihood of that happening is low enough that we just send the lpa UID and do a mass update based on that.